cirq/sim/simulator.py

```diff
@@ -605,7 +605,7 @@ def density_matrix(self, indices: Iterable[int] = None) -> np.ndarray:
                             \end{bmatrix}
```

Comment:
You need to update the docstring to replace instances of `indices` with `qubits`.

Reply:
fixed it